frontend/src/components/AppHeader.vue

```diff
@@ -316,11 +296,26 @@ export default {
               rel: "noopener",
             },
             {
-              text: "Contactez-nous",
-              to: { name: "ContactPage" },
+              text: "Mesures de notre impact",
+              to: { name: "ImpactMeasuresPage" },
+            },
+            {
+              text: "Indicateurs de résultats de la plateforme",
```

Comment:
Est-ce qu'on peut trouver un titre plus court ? Sur mon écran je vois un ... après, p-e pour l'icon open-in-new

`Résultats de la plateforme` ?

![Screenshot 2023-09-29 at 12-07-48 ma cantine](https://github.com/betagouv/ma-cantine/assets/9282816/01fa9856-163f-4266-8991-7d50e8d97cd0)

Reply:
Changement : https://github.com/betagouv/ma-cantine/pull/3005/commits/5bcbf82bb2e1adc03bd48ce8321f607d6f8c01ea

Reply:
hm, j'ai toujours le pb de `...` sur mon écran :/

Reply:
Fix : https://github.com/betagouv/ma-cantine/pull/3005/commits/2ee8da83b5068e614653ccffb8c79c85c7679a39